azure-pipelines.yaml

```diff
@@ -42,8 +42,30 @@ steps:
             done
```

Comment:
@muradsater Would it be possible to change this to:
```
echo "##vso[task.setvariable variable=INPUT_$CONFIGURATION;]:$VALUE"
```
...to avoid the "duplication" below?

Reply:
I don't think so. This only sets variables from the config yaml files. The sample pipeline are using environment variables.  But I don't have time to test it now

Reply:
@muradsater so you are right - but for the wrong reason:
# Variable naming restrictions
User-defined variables can consist of letters, numbers, ., and _ characters. Don't use variable prefixes that are reserved by the system. These are: endpoint, input, secret, and securefile. Any variable that begins with one of these strings (regardless of capitalization) will not be available to your tasks and scripts.

...so any variables starting with "input" will be removed, which is also what I see when I test it out. I think I'll modify the function action accommodate the differences 👌